lib/iris/tests/stock.py

```diff
@@ -35,6 +35,53 @@
 from iris.coord_systems import GeogCS, RotatedGeogCS
 
 
+def geodetic(shape, with_bounds=True, xlim=None, ylim=None, circular=False):
+    """
+    Return a global lat-lon cube of specified shape.
+
+    """
+    if xlim is None:
+        xlim = (-180, 180)
+    if ylim is None:
+        ylim = (-90, 90)
+    shape = np.array(shape)
+    assert shape.ndim == 1
+    assert shape.shape[0] == 2
```

Comment:
I find the conversion to an array and subsequent assertions somewhat confusing ... how about just: `y_size, x_size = shape`?